dns/dyndns/src/www/services_dyndns_edit.php

```diff
@@ -382,6 +397,12 @@ function is_dyndns_username($uname)
                       <input name="resourceid" type="text" id="resourceid" value="<?= $pconfig['resourceid'] ?>" />
                       <div class="hidden" data-for="help_for_resourceid">
                         <?= gettext("Enter the resource id of the DNS Zone in Azure.") ?>
+                  <tr class="opt_field type_netcup">
+                    <td><a id="help_for_resourceid" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?= gettext("customer Id") ?></td>
+                    <td>
+                      <input name="resourceid" type="text" id="resourceid" value="<?= $pconfig['resourceid'] ?>" />
```

Comment:
why is there a second resourceid input now? :)

Reply:
Whops :wink:
I think I tried to change the field labels.  Maybe I should merge the two html rows and only differentiate on gettext as "context-sensitive" user hint, then. Something like:
````
<tr class="opt_field type_azure type_netcup">
  [...]
     <div class="hidden" data-for="help_for_resourceid">
       <br class="opt_field type_azure"/><?= gettext("Enter the resource id of the DNS Zone in Azure.") ?>
       <br class="opt_field type_netcup"/><?= gettext("Enter your netcup customer number for the account hosting the dns zone.") ?>
````
Would this work for you? I like the idea of showing specific user help linked to the current input form.

Reply:
> why is there a second resourceid input now? :)

I fixed the duplicate input field in latest commit.